README.md

```diff
@@ -42,30 +45,30 @@ We expect a `results.csv` file following this format, ordered by `Continuity` (d
 The field `Continuity` is expressed in days.
 
 ```
-Worker, Continuity
-1435, 3
-135,  2
-456,  1
+Worker,Continuity
+1435,3
+135,2
+456,1
 ```
 
 
 ## Bonus Points
 
-We don't expect your solution to use sophisticated paradigms or frameworks. However, you can spend as much time you want to demonstrate your skills. You can, for example, implement your application in a stream-oriented fashion, or document how you would implement this in addition to your batch-oriented implementation.
+We don't expect your solution to use sophisticated paradigms or frameworks. However, you can spend as much time you want to demonstrate your skills. 
+You could, for example, implement some tests to ensure your output is correct. 
 
 
 ## Open questions
 
 As part of your submission, please answer the following questions in a `readme.txt` file.
 
 1. What is the complexity of your algorithm?
-2. If you were to review someone else's code, what would you pay attention to?
-3. We provided a CSV file for conveniency, but in a production environment, what format or database would you recommend instead for this use-case (and why)?
-4. Is there anything else you would implement differently for a large-scale application in production?
+2. We provided a CSV file for convenience, but in a production environment, what format or database would you recommend instead for this use-case (and why)?
+3. Is there anything else you would implement differently for a large-scale application in production?
 
 
 ## Submission
 
 Please write your application in either Python, Java, Scala or Kotlin, with a runnable shell script named `run-app.sh` to launch your application.
-Please put your results in `result.csv` and submit all the files in a tarball named as follows: `syft-dataeng-<firstname>-<lastname>.tar.gz`.
-It's recommended to spend approximately 1 hour on this challenge. However, you are free to elaborate as much as you want to show us your ability to deliver great software!
+Please put your results in `result.csv` and submit all the files in a zip file named as follows: `indeedflex-dataeng-<firstname>-<lastname>.zip`.
```

Comment:
Makes me sad, but I reckon this will help candidates who're stuck with the gmail spam filter.

Reply:
Happy to change it back, but thought this might be more straightforward?

Reply:
Yeah I don't mind, I think I'm just emotionally attached to anything POSIX, and enjoy the thought of having candidates thinking about the process to generate a tarball.